libs/wingc/src/lsp/completions.rs

```diff
@@ -102,29 +24,428 @@ pub unsafe extern "C" fn wingc_on_completion(ptr: u32, len: u32) -> u64 {
 		WASM_RETURN_ERROR
 	}
 }
+
 pub fn on_completion(params: lsp_types::CompletionParams) -> CompletionResponse {
```

Comment:
(for my curiosity) are there any other interesting params / will any of them be relevant to us in the future?

Reply:
Not sure what you mean. For completion this is the whole thing, or are you talking about the rest of the LSP stuff? If so, yeah there are tons. The tough part is prioritization.